src/main/scala/akka/persistence/cassandra/query/scaladsl/CassandraReadJournal.scala

```diff
@@ -358,7 +365,7 @@ class CassandraReadJournal(system: ExtendedActorSystem, config: Config)
       None,
       s"currentEventsByPersistenceId-$persistenceId"
     )
-      .map(r => toEventEnvelope(r, r.sequenceNr))
+      .mapConcat(r => toEventEnvelopes(r, r.sequenceNr))
 
   private[cassandra] def eventsByPersistenceId(
```

Comment:
please add a comment to this method that it is used by the `CassandraRecovery` replay and does not apply event adapters, since that is performed by the ordinary replay.